core/field_angle.js

```diff
@@ -57,7 +57,7 @@ goog.inherits(Blockly.FieldAngle, Blockly.FieldTextInput);
  * Round angles to the nearest 15 degrees when using mouse.
  * Set to 0 to disable rounding.
  */
-Blockly.FieldAngle.ROUND = 15;
+Blockly.FieldAngle.ROUND = 1;
```

Comment:
Why did you change this from 15?  I'm guessing it was while debugging, but it should stay at 15 as the default.